CONTRIBUTING.md

```diff
@@ -1,93 +1,27 @@
-# Contributing
+# Contributing to metsrw
 
-Thank you for considering a contribution to the METS Reader/Writer!
-This document outlines the change submission process for metsrw, along with our
-standards for new code contributions. Following these guidelines helps us assess
-your changes faster and makes it easier for us to merge you submission!
-
-There are many ways to contribute: providing feedback on the [design & API](https://github.com/artefactual-labs/mets-reader-writer/wiki),
-submitting bug reports, or writing code which can be incorporated into metsrw
-itself.
-
-## Table of contents
+Thank you for your interest in making a contribution to metsrw.
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
```

Comment:
I think we can remove the `doctoc` hook from this project too since there's only one additional section for now.